mmv1/third_party/terraform/.teamcity/PERFORMING_TASKS_IN_TEAMCITY.md

```diff
@@ -0,0 +1,115 @@
+# How to Use TeamCity
```

Comment:
Exciting to have this documented in code! And alongside other TeamCity-related things!